src/tables/StackedTable.tsx

```diff
@@ -34,17 +34,36 @@ const StackedTable = (props: StackedTableProps) => {
         content: (
           <div
             className={`stacked-table-cell-container ${
-              props.headersHiddenDesktop?.includes(item) && "md:hidden"
+              props.headersHiddenDesktop?.includes(item) ? "md:hidden" : ""
             }`}
           >
-            <span className={"stacked-table-cell"}>{dataRow[item].cellText}</span>
-            <span
-              className={`stacked-table-subtext  ${
-                dataRow[item].hideSubTextMobile && "hidden md:block"
-              } `}
-            >
-              {dataRow[item].cellSubText}
-            </span>
+            {Array.isArray(dataRow[item]) ? (
+              dataRow[item].map((item: StackedTableRow, index: number) => {
+                return (
+                  <div key={index}>
+                    <span className={"stacked-table-cell"}>{item.cellText}</span>
+                    <span
+                      className={`stacked-table-subtext  ${
+                        item.hideSubTextMobile ? "hidden md:block" : ""
+                      } `}
+                    >
+                      {item.cellSubText}
```

Comment:
Could we pull this JSX into a lil fxn to reduce duplication?

Reply:
absolutely!